int64 call
---
int64[9] boardcase
